smithy-aws-apigateway-openapi/src/test/resources/software/amazon/smithy/aws/apigateway/openapi/authorizers.json

```diff
@@ -5,18 +5,14 @@
             "type": "service",
             "version": "2018-03-17",
             "traits": {
-                "smithy.api#protocols": [
-                    {
-                        "name": "aws.rest-json",
-                        "auth": [
-                            "aws.v4"
-                        ]
-                    }
-                ],
+                "aws.protocols#restJson1": {},
+                "aws.auth#sigv4": {
+                    "name": "someservice"
```

Comment:
"name" is required (at least for now). This makes the sigv4 signing name much more explicit.